workload_generator/src/workload_gen.py

```diff
@@ -123,75 +131,74 @@ def sendWorkload(self, user, pid):
 
             if command == BUY:
                 (stockSymbol, amount) = (requestInfo[2], float(requestInfo[3].strip("\n")))
-                self.performRequest(pid, transactionNumber, command, user, stockSymbol, amount)
+                self.performRequest(url1, transactionNumber, command, user, stockSymbol, amount)
```

Comment:
replaced pid here with url for the specific request to route request to the correct url. 


for example either
```
http://localhost:8000/api/stocks/
http://localhost:8000/api/accounts/
```